src/components/datepicker/calendar.js

```diff
@@ -58,6 +58,7 @@
       scope: {
         minDate: '=mdMinDate',
         maxDate: '=mdMaxDate',
+        isDateEnabled: '=mdDateEnabled',
```

Comment:
I'd go with md-date-filter but it somehow confuses me what to return to make date available (maybe it's just me).